solarwinds_apm/trace/serviceentry_processor.py

```diff
@@ -78,12 +78,15 @@ def on_start(
         # Calculate non-custom txn name for entry span if we can retrieve the URL
         # or serverless name. Otherwise, use the span's name
         pool = get_transaction_name_pool()
+
+        sw_apm_txn_name = os.environ.get("SW_APM_TRANSACTION_NAME", None)
```

Comment:
Since is "the" place that default transaction naming happens now, please could we update the docstring (L68) to something like this:

```
        """Calculates default transaction name for span and metrics following this order
        of decreasing precedence, truncated to 255 char:

        1. SW_APM_TRANSACTION_NAME
        2. Any instrumentor-set span attributes for FaaS
        3. AWS_LAMBDA_FUNCTION_NAME
        4. Any instrumentor-set span attributes for HTTP
        5. Span name (default)
        6. "unknown" (when limit reached)

        Also caches entry span at its trace ID for custom transaction naming with SDK.
        """
```

Reply:
Done.
Corrected 
6. "other" (when the transaction name pool limit reached)